feel/feeldiscr/functionspace.hpp

```diff
@@ -4313,38 +4313,55 @@ public std::enable_shared_from_this<FunctionSpace<A0,A1,A2,A3,A4> >
                    size_type mesh_components = MESH_RENUMBER | MESH_CHECK,
                    periodicity_type  periodicity = periodicity_type(),
                    worldscomm_ptr_t const& _worldsComm = Environment::worldsComm(nSpaces),
-                   std::vector<bool> extendedDofTable = std::vector<bool>(nSpaces,false) )
+                   std::vector<bool> extendedDofTable = std::vector<bool>(nSpaces,false),
+                   const std::string& name = functionSpaceDefaultInstanceName() )
```

Comment:
I think it misses to specify the name in the static function FunctionSpace::New or pch,pchv, ... ( methods used by users for build a functionspace)